src/couch_views/test/couch_views_indexer_test.erl

```diff
@@ -610,6 +613,23 @@ index_can_recover_from_crash(Db) ->
     ], Out).
 
 
+handle_acquire_map_context_error(_) ->
+    meck:new(mock_language_server, [non_strict]),
+    config:set("couch_eval.languages", ?QUERY_SERVER_LANG_STRING,
+        atom_to_list(mock_language_server)),
+    meck:expect(mock_language_server, acquire_map_context,
```

Comment:
Minor nit: usually the formatting for fun would look like:

```
meck:expect(mock_language_server, acquire_map_context, fun(_) ->
    {error, foo_error}
end)
```

Reply:
Done in 71f2765